base/changes.txt

```diff
@@ -6,6 +6,11 @@ completeness or accuracy and it contains some references to files that
 are not part of the distribution.
 ================================================================================
 
+2021-09-28 Phelype Oleinik  <phelype.oleinik@latex-project.org>
+
+	* ltcmdhooks.dtx (subsection{Patching commands}):
+	Make patching commands a global operation (gh/674).
+
```

Comment:
would mention what is patched:

Make patching commands with generic cmd hooks a global operation (gh/674)

it is not that each kind of patching is suddenly global

Reply:
Make patching of commands . . . 

(more accurate!)

Reply:
Also agreeing with Frank's change.